### DISCUSSION TITLE
AKS compatibility

### DISCUSSION THREAD

**Discussion by wedge76** [kube-green/kube-green#292] [Q&A] [answered]
This is a cool project, I really like it!

1. We are hosting in AKS, is kube-green also compatible with this cloud?
2. For my understanding, why is the cert-manager needed?

**Comment by davidebianchi** [comment, marked as answer]
Hi @wedge76! Thank you!

If you use it, would you like to add your org to the adopters list (https://github.com/kube-green/kube-green.github.io/blob/main/CONTRIBUTING.md#add-your-organization-to-adopters)?

1. Yes, it should be compatible with also with AKS, it is required with all K8s distribution. If we found some issues or quirks to the installation on a cloud provider, we write it here: https://kube-green.dev/docs/install/#cloud-compatibility
2. cert-manager is suggested to provide certificates for the webhook server. It is possible also to do it in other ways (so, without the use of cert-manager), but it's necessary to put the certificates in the desired location. kube-green is based on kubebuilder scaffold, and [here](https://book.kubebuilder.io/cronjob-tutorial/cert-manager.html) you can find the documentation about cert-manager.

**Reply by wedge76** [reply]
Hi @davidebianchi 
thanks for you quick response. I'm definitely going to try it out on AKS and let you know.
Kind regards

**Reply by wedge76** [reply]
Hi @davidebianchi 
I can confirm, it also works with AKS. 
Kind regards
Peter
